docs/4.3/enterprise/workflow/ssh_approval_jira_server.md

```diff
@@ -150,6 +163,15 @@ You should be able to run the Teleport plugin now!
 
 ```bash
 teleport-jira start
+INFO   Starting Teleport Access JIRAbot 0.1.0-alpha.3:teleport-jira-v0.1.0-alpha.3-0-gea1ef8e jira/app.go:74
+# DEBU   Checking Teleport server version jira/app.go:150
+# DEBU   Starting JIRA API health check... jira/app.go:111
+# DEBU   Checking out JIRA project... jira/bot.go:145
+# DEBU   Found project "TEL1": "Tel-kb" jira/bot.go:150
+# DEBU   Checking out JIRA project permissions... jira/bot.go:152
+# DEBU   JIRA API health check finished ok jira/app.go:117
+# DEBU   Starting secure HTTPS server on 66.220.18.10:8081 utils/http.go:235
```

Comment:
Might want to obfuscate this IP if the server is going to stay around for any length of time